README.md

```diff
@@ -223,7 +223,7 @@ the container for you - you'll need to do that yourself.
 Debugging
 ---------
 
-You can run the server manually in foreground mode by running `SCCACHE_START_SERVER=1 SCCACHE_NO_DAEMON=1 sccache`, and send logging to stderr by setting the `RUST_LOG` environment variable, the format of which is described in more detail in the [env_logger](https://docs.rs/env_logger/0.5.3/env_logger/#enabling-logging) documentation.
+You can run the server manually in foreground mode by running `SCCACHE_START_SERVER=1 SCCACHE_NO_DAEMON=1 sccache`, and send logging to stderr by setting the [`RUST_LOG` environment variable](https://docs.rs/env_logger/0.7.1/env_logger/#enabling-logging) e.g. `RUST_LOG=debug`.
```

Comment:
I think we should have an example and a pointer to further documentation; could you make that change, please?

Reply:
What would be an example of `SCCACHE_ERROR_LOG`? Is this right?

    SCCACHE_ERROR_LOG=/tmp/sccache_log.txt RUST_LOG=debug sccache

Reply:
Yup!